llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1774,6 +1774,12 @@ let Predicates = [IsRV64], hasSideEffects = 0, mayLoad = 0, mayStore = 0,
 def PseudoZEXT_W : Pseudo<(outs GPR:$rd), (ins GPR:$rs), [], "zext.w", "$rd, $rs">;
 } // Predicates = [IsRV64], ...
 
+let Predicates = [IsRV64] in
```

Comment:
Add the new patterns to RISCVGISel.td for now since they aren't needed by SelectionDAG.,